### DISCUSSION TITLE
Some puzzle about the function

### DISCUSSION THREAD

**Discussion by TristanY126** [NatLabRockies/floris#722] [Q&A]
I am a freshman both in FLORIS and windfarm research. Here, I have some questions about the usage and functions of FLORIS.

First, it is puzzle for me that the distance "D" in the FLORIS. The chatgpt tell me that D=126meters. Why?

Second, I can't understand why the reference wind height is -1?

Third, I want the FLORIS to deal with the daily data (CSV) directly to find out the power generation changes day by day in 1 year. Can anyone help me?

Thanks in advance for anyone's assiatance.

**Comment by rafmudaf** [comment]
Hi @TristanY126 thanks for getting in touch. So I know how to help, can you describe what you've read or studied to understand the FLORIS software? Unfortunately, I suggest that you do not rely on ChatGPT for understanding this tool as FLORIS is probably changing faster than ChatGPT is retrained. Also, you'll find a lot of value in building your intuition of the concepts so that you can use them creatively going forward.
